2.5/test/run

```diff
@@ -181,6 +181,27 @@ for server in ${WEB_SERVERS[@]}; do
 
   info "All tests for the ${server}-test-app finished successfully."
   cleanup ${server}
+
 done
 
+# test from dockerfile examples
+pushd ${test_dir}
```

Comment:
Could you please implement the tests in the same way as the other ones (wrapped in a method, for one)?  This seems way more chaotic.